### PR TITLE
metaImport needs a -n option

### LINKED ISSUE #1455: metaImport needs a -n option
Discussed with @pieterlukasse and thought about this some more last night.

I think the metaImport script needs a -n (--no_portal_checks) option.  This option already exists in validateData.py, we just need to pass it along.

That way a new user who is following the installation steps on the wiki can successfully import data without having a portal instance already running.  This is important because in the wiki docs, we suggest that people get the entire database set up first, and then deploy the web app.  Without a -n option, new installs are blocked from importing data.

@pieterlukasse suggested that we use cbioportalImporter.py, which provides this functionality.  But, I would much rather introduce the right scripts up-front, e.g. introduce others to the validator and metaImport, so that they can see how it works up-front.

Since @priti88 is out, I will endeavor to try to fix on my own today.  Otherwise, may call upon @ersinciftci.

### PR DESCRIPTION
**Pull request by ersinciftci**
# What? Why?
Fix #1455  

# Checks
- [ ] Runs on Heroku.
- [ ] Single commit and [No merge commits](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [ ] Make sure your commit messages end with a Signed-off-by string (this line
  can be automatically added by git if you run the `git-commit` command with
  the `-s` option)
- [ ] If this is a feature, the PR is to rc. If this is a bug fix, the PR is to
  hotfix.

# Notify reviewers
If your pull request involves changes to an existing file, one or more of the
people that edited before you might be good candidates to review it. Please use
`git blame <filename>` to determine that and notify them here. Otherwise notify
everybody in the core team:
@ecerami @pieterlukasse @fedde-s

### CODE REVIEW

**Comment by fedde-s** [comment]
I think adding this option is a bad idea as I mentioned in issue #1455, but if you really do want it, it should be mutually exclusive with `-u` and `-p` as it is in [validateData.py](https://github.com/cBioPortal/cbioportal/blob/rc/core/src/main/scripts/importer/validateData.py#L2579)

**Comment by fedde-s** [comment]
As I mentioned in the issue, I left out the option to load data after only partial validation by design and find adding it a dangerous idea, *especially* if it’s for teaching people what is the right way to do it . If people insist on skipping validation (the old way), there’s still the `cbioportalImporter.py` script that’s used by `metaImport.py` in the background.

File: core/src/main/scripts/importer/metaImport.py [hunk 1]
```diff
@@ -52,6 +52,10 @@ def interface():
                                    help='Path to a directory of cBioPortal '
                                         'info files to be used instead of '
                                         'contacting the web API')
+    portal_mode_group.add_argument('-n', '--no_portal_checks', default=False,
+                                       action='store_true',
+                                       help='Skip tests requiring information '
+                                            'from the cBioPortal installation')
```
Comment (pieterlukasse) [inline core/src/main/scripts/importer/metaImport.py]: I would change the description here to `hep='start the loading process with minimal validation, skipping tests that require information from the cBioPortal installation. Please note that the loading process is not guaranteed to succeed in this case'`

**Comment by ecerami** [comment]
This will likely be a temporary fix for ~2-3 weeks until @priti88 is back. I have logged my preferred solution as #1466.

**Pull request closed without merging**